webpack.config.js

```diff
@@ -0,0 +1,123 @@
+"use strict";
+
+const path = require('path')
+const HtmlWebPackPlugin = require('html-webpack-plugin')
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin')
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
+const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin')
+const PACKAGE = require('./package.json')
+const version = PACKAGE.webpack_bundle_version
+const CleanWebpackPlugin = require('clean-webpack-plugin')
+const project_config = require('./config.js')
```

Comment:
It doesn't look like any of these are dependent on each other, so let's alphabetize for ease of reading.